### ISSUE TITLE
ImportError: cannot import name App

### ISSUE THREAD

**Issue by lisahale** [miguelgrinberg/flasky#338]
Hi there, I have been looking for an resolution for hours.... 

I am getting this error 
File "/var/www/MyApptool/flasktool.wsgi", line 5, in <module>
from MyApptool import rfc as application
 ImportError: cannot import name MyApptool

Here is my wsgi file:

#!/opt/MyApptool/bin/python3
import os
import sys

from MyApptool import MyApptool as application
logging.basicConfig(stream=sys.stderr)
sys.path.insert(0,"/var/www/MyApptool/")

Thank you inadvance

**Comment by miguelgrinberg** [comment]
You are not providing enough information. Do you have a `MyApptool` module or package in the `/var/www/MyApptool/` directory? Your Python interpreter says you don't, yet that is what you are importing.

**Comment by lisahale** [comment]
Thank you for responding... Yes, I have a MyAppTool Module and the __init__.py file looks like this
def hello():
	return 'Hello world!'
def add(x,y):
	return x+y

**Comment by lisahale** [comment]
Plus, here is the other error that i am getting....

[cgi:error] [pid 48097] [client 24.93.67.155:15671] End of script output before headers: run.py

**Comment by miguelgrinberg** [comment]
Are you following the exercises in the book? None of the code you shown appears to come from the book. Unfortunately I can't really help you. You are trying to import something that does not exist, or is located in a different place. You need to review the Python documentation regarding importing packages and modules, and then figure out what you need to fix to make those imports work, either move the modules or packages to the proper place, or else change the imports so that they import from the right location.

**Comment by lisahale** [comment]
No, I am trying to host an Flask/python application on a Apache cento7 server. Ok, thank you

**Comment by lisahale** [comment]
Ok, I have another question... in the WSGI file should the sys.path.insert(0,"/var/www/MyApptool/") point to the location of the modules? or should i add that to the httpd.conf?

Thanks in advance

**Comment by miguelgrinberg** [comment]
@lisahale I don't use apache, my guess is that you can configure your path directly in the Python code or in the config file. It can probably be done both ways, but I suggest you ask people who use this method of deployment regularly.
